### ISSUE TITLE
extend blog webmentions feature to all web sites

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy#420]
...ie not just blogger, tumblr, and wp.com. [inspired by IRC today.](http://indiewebcamp.com/irc/2015-06-28#t1435513591228)

the vast majority of this is done. we just need a text box next to the blog providers [on the front page](https://www.brid.gy/) to enter your domain, maybe indieauth or check for a backlink to prove you own the domain (not strictly necessary since anyone can send a webmention for any source and target), and maybe tweak a few other things that come up. surprisingly little work required.

**Comment by diplix** [comment]
i like that.

**Comment by miklb** [comment]
+1

**Comment by snarfed** [comment]
@aaronpk's telegraph now does this! https://telegraph.p3k.io/superfeedr . makes it a bit less of a priority to do it in bridgy too.

**Comment by miklb** [comment]
@snarfed hmm. I thought that was for being able to receive webmentions from sites that don't technically support them, not for being able to send them if you are on a platform that doesn't have a method for sending.

**Comment by snarfed** [comment]
> hmm. I thought that was for being able to receive webmentions from sites that don't technically support them

right! that's what this issue is for.

> for being able to send them if you are on a platform that doesn't have a method for sending.

that's what bridgy's existing blog feature is! it sends and receives wms for blogs on tumblr, blogger, and wordpress.com. https://brid.gy/about#blogs

**Comment by snarfed** [comment]
...oh, do you mean extending the blog feature to sites that aren't on tumblr/blogger/wp.com? you're right, we could definitely do that too! at least for sending. i'd probably encourage people to use a native plugin in their CMS, for that though, or telegraph itself, since that's what it's for.

**Comment by miklb** [comment]
oh. OK :-) I thought it was the opposite. Allow users who aren't on tumblr, blogger or wp.com to be able to  easily send webmentions (i.e. Jekyll sites on GitHub pages.)

**Comment by miklb** [comment]
@snarfed yes, to follow up, Jekyll sites on GH pages was the specific use case I'm thinking about. There isn't a "simple" way for those users to send webmentions. Inserting some js to receive them isn't an issue. I'm used a convoluted method of a ruby plugin and building via Travis CI and then pushing back to GitHub in order to send. Not for the faint of heart or beginning user.

I'm currently working on a Jekyll "theme" that a new user could clone, edit a few lines of the config and be up & running with an indieweb site fairly quickly and easily. Having a solution for also sending would be 👍

Just a thought.

**Comment by snarfed** [comment]
@miklb definitely! sounds like a good telegraph feature request, especially since it already has some superfeedr integration. cc @aaronpk.

**Comment by voxpelli** [comment]
A h-feed to Webmention pionger would be a great extension of Telegraph or similar. It's something I need for my Jekyll blog as well and have been thinking of building myself if nothing else. Consuming a Pubsubhubbub h-feed/rss and sending Webmentions for all relevant links in new/updated posts in there.

**Comment by snarfed** [comment]
@aaronpk's telegraph now does this! https://telegraph.p3k.io/superfeedr

**Comment by snarfed** [comment]
Also [webmention.app](https://webmention.app/) ([docs](https://webmention.app/docs#using-ifttt-to-trigger-checks)) and [Pushl](https://github.com/PlaidWeb/Pushl).
